### DISCUSSION TITLE
v1.4.2

### DISCUSSION THREAD

**Discussion by Chizaruu** [Chizaruu/com.tsk.ide.vscode#56] [Announcements]
- Restrict Shell Compatibility for Project Generation's Dotnet Build (Mac/Linux) 

**Full Changelog**: https://github.com/Chizaruu/com.tsk.ide.vscode/compare/1.4.1...1.4.2

<hr /><em>This discussion was created from the release <a href='https://github.com/Chizaruu/com.tsk.ide.vscode/releases/tag/1.4.2'>v1.4.2</a>.</em>

**Comment by Jaakk0S** [comment]
Hey! Do you think there's a way to switch off the Roslyn analyzers? I think they're somehow auto-included. I switched them off from both VScode settings and omnisharp.json but I am still getting suggestions. I can filter these out from the Problems tab but they still keep the squigglies and other annotations that I don't want to see.

**Reply by Chizaruu** [reply]
Ah, yeah. Assets Folder related asmdefs get the Unity Analysers nuget reference added to their csproj.

Could you create an issue so I can be reminded to tweak the code to address this? :)
